x/dsmr/p2p.go

```diff
@@ -127,10 +128,18 @@ func (c ChunkSignatureRequestVerifier[T]) Verify(
 		}
 	}
 
+	// verify that the signature is a valid signature over the unsigned chunk.
 	if err := c.verifier.Verify(chunk); err != nil {
 		return ErrInvalidChunk
 	}
 
+	// TODO:
+	// check if the expiry of this chunk isn't in the past or too far into the future.
+
+	// TODO:
+	// check if the producer was expected to produce this chunk.
+
+	// check to see if this chunk was already accepted.
```

Comment:
Can these comments be moved into the verifier?

From p2p perspective, it should only need to know that it must verify the chunk in some context, but doesn't need to know what verification takes place.

Reply:
Should we add a TODO for rate limiting chunk production from the given producer?